sorc/global_cycle.fd/cycle.f90

```diff
@@ -444,6 +452,10 @@ SUBROUTINE SFCDRV(LUGB, IDIM,JDIM,LENSFC,LSOIL,DELTSFC,  &
  ENDIF
 
 IF (DO_LNDINC) THEN
+   IF (FRAC_GRID) THEN
```

Comment:
@ClaraDraper-NOAA - looks like I stop processing when DO_LNDINC is chosen with fractional grids. Do you want to fix this now or in a subsequent issue?

Reply:
Sounds good.